Airports/EGGP/Sector.txt

```diff
@@ -1,21 +1,21 @@
 CIRCLE_SECTORLINE:Only GPGND:EGGP:1.5
-DISPLAY:Scottish PC West:Only GPGND 1:Only GPGND 2
-DISPLAY:Scottish PC East:Only GPGND 1:Only GPGND 2
+DISPLAY:Wallasey:Only GPGND 1:Only GPGND 2
+DISPLAY:North:Only GPGND 1:Only GPGND 2
```

Comment:
Should we display GND/TWR/APP rings to Stafa as well, given that KEGUN/TIPOD traffic from the SE routes this way? Probably logical if we show to North

![image](https://user-images.githubusercontent.com/12017284/62325456-98ef4f80-b4a3-11e9-81da-62c0fada3361.png)